neptune/new/exceptions.py

```diff
@@ -45,6 +45,28 @@ class MetadataInconsistency(NeptuneException):
     pass
 
 
+class MissingAttributeException(NeptuneException):
+    """Raised when get-like action is called on `Handler`, instead of on `Attribute`."""
+
+    def __init__(self):
+        message = """
+{h1}
+----MissingAttributeException---------------------------------------------------
+{end}
+Neptune Client Library can't access given `Attribute`.
+
+Probably `Attribute` wasn't synced with backend API yet.
+You can sync it by either calling:
+* `.wait()`
```

Comment:
It won't work in this scenario. Only sync() should be hinted here.

Reply:
ok